collections-csharp/ConcurrentQueueInCSharp/ConcurrentQueueInCSharp.Console/OrderMessageBus.cs

```diff
@@ -0,0 +1,30 @@
+﻿using ConcurrentQueueInCSharp.Models;
+using System.Collections.Concurrent;
+
+namespace ConcurrentQueueInCSharp
+{
+    public class OrderMessageBus
+    {
+        private readonly ConcurrentQueue<Order> _queue = new ConcurrentQueue<Order>();
+
+        public int Count
+        {
+            get
+            {
+                return _queue.Count;
+            }
+        }
+
+        public void Add(Order order)
```

Comment:
Since we are allowing null as an input (for testing purposes), we should mark this as nullable

Reply:
Good point.